### PR TITLE
Solaris: fix stdio issue for x64 OS.

### PR DESCRIPTION
**Pull request by nykytenko**
Fix stdio issue for x64 Solaris (SmartOS x64) by add correct struct _iobuf.

### CODE REVIEW

File: src/core/stdc/stdio.d [hunk 1]
```diff
@@ -401,27 +401,50 @@ else version( FreeBSD )
     ///
     alias shared(__sFILE) FILE;
 }
-else version (Solaris)
+else version( Solaris )
```
Comment (DmitryOlshansky) [inline src/core/stdc/stdio.d]: Revert to original style

**Comment by DmitryOlshansky** [comment]
Otherwise looks good to me

**Comment by nykytenko** [comment]
Done.

**Comment by DmitryOlshansky** [comment]
Auto-merge toggled on

**Pull request merged**